support/index.markdown

```diff
@@ -12,11 +12,11 @@ title: Support
 
   * First, check out the list of [Frequently Asked Questions](/documentation/faqs) to see if your question has been answered already.
 
-### Mailing List
+### Find Similar Questions
 
-  * Search the [Mailing List Archive](https://groups.google.com/forum/#%21forum/moveit-users) for a similar question to yours
-
-  * [Join the MoveIt! users mailing list](https://groups.google.com/forum/#!forum/moveit-users/join) (moveit-users _at_ googlegroups _dot_ com)
+  * Web-search for a similar question to yours. Search by keyword, error messages etc.
```

Comment:
I don't think this item is required.
The next two items point to concrete positions and I believe this is perfectly enough.